Initialize-SharedInfrastructureDeployment.ps1

```diff
@@ -68,18 +68,57 @@ try {
     $ResourceGroupList = [System.Collections.ArrayList]::new(@($ManagementResourceGroupName))
     $ResourceGroupList.AddRange(@($EnvironmentNames | ForEach-Object { "das-$_-shared-rg".ToLower() }))
 
+    $Tags = @{
```

Comment:
Should we really be duplicating tagging logic when we have an existing script for it?

Reply:
Spoken to you in person about this. I agree about the duplication but feel like we should wait for @dalemorson to merge the infra scripts in to das-platform-automation rather than consume the legacy nuget package.